src/Traits/HasPermissions.php

```diff
@@ -52,7 +52,7 @@ public function permissions(): MorphToMany
     /**
      * Scope the model query to certain permissions only.
      *
-     * @param \Illuminate\Database\Eloquent\Builder $query
+     * @param \Illuminate\Database\Eloquent\Builder                                               $query
```

Comment:
While I too like having PHPStorm align variable names when reformatting, one thing it does poorly is this scenario when it shoves in 30+ spaces and actually creates more confusion instead of improving readability.
So in this package we opt to turn off that "alignment" setting. 
Please remove the extra spaces added in docblocks.

Reply:
Apologies - tried very hard not to press my auto-format shortcut while doing this, apparently, I failed.

Reply:
No worries - I'm always catching myself doing that too!